src/frontend/flask_server.py

```diff
@@ -496,7 +496,7 @@ def format_currency(int_amount):
     os.environ.get('CONTACTS_API_ADDR'))
 APP.config['PUBLIC_KEY'] = open(os.environ.get('PUB_KEY_PATH'), 'r').read()
 APP.config['LOCAL_ROUTING'] = os.getenv('LOCAL_ROUTING_NUM')
-APP.config['BACKEND_TIMEOUT'] = 3  # timeout in seconds for calls to the backend
+APP.config['BACKEND_TIMEOUT'] = 4  # timeout in seconds for calls to the backend
```

Comment:
Was the timeout increase necessary because of the generate username changes?  Or was it needed before this change?

Reply:
It seems that it is necessary after the generating of username change, I think it is the overhead of the for loop taking a little longer (couple milliseconds) more than expected.

Reply:
sg, thanks for clearing that up.  LGTM